db/migrate/20230622151208_remove_existing_segmented_users.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveExistingSegmentedUsers < ActiveRecord::Migration[7.0]
+  def up
+    SegmentedUser.delete_all
```

Comment:
I'm fine with the de-dupe query, but just for having options: we could also tell audience segments to refresh themselves, if we don't want to wait until the overnight cron.

```suggestion
    SegmentedUser.delete_all
    AudienceSegment.find_each &:refresh!
```